charts/celeborn/templates/worker-statefulset.yaml

```diff
@@ -109,9 +108,15 @@ spec:
         - configMap:
             name: {{ .Values.configmap }}
           name: {{ include "celeborn.fullname" . }}-volume
-        {{- range $index, $dir := $dirs }}
-        - hostPath:
-            path: {{ $dir }}/worker
+        {{- range $index, $volume := .Values.volumes.worker }}
+        {{- if eq "emptyDir" $volume.type }}
+        - name: celeborn-worker-vol-{{ $index }}
+          emptyDir:
+            sizeLimit: {{ $volume.size }}
+        {{- else if eq "hostPath" $volume.type }}
+        - name: celeborn-worker-vol-{{ $index }}
+          hostPath:
+            path: {{ $volume.path }}/worker
             type: DirectoryOrCreate
-          name: vol-{{ $index }}
-        {{- end }} 
+        {{- end }}
```

Comment:
can we add an else branch to fail fast to guard other types?

Reply:
Sure, fixed.